bundles/org.openhab.ui/web/src/assets/definitions/metadata/namespaces.js

```diff
@@ -12,5 +12,6 @@ export default [
   { name: 'voiceSystem', label: 'Voice System' },
   { name: 'alexa', label: 'Amazon Alexa' },
   { name: 'homekit', label: 'Apple HomeKit' },
-  { name: 'ga', label: 'Google Assistant' }
+  { name: 'ga', label: 'Google Assistant' },
+  { name: 'link_to_more', label: 'Android App: Link to More' }
```

Comment:
I'd keep the "link to more" away from user-facing strings. When there's some other purpose for this metadata, we can rename it to "link to more", but currently there isn't.

```suggestion
  { name: 'link_to_more', label: 'Android App: Device Controls' }
```